docs/components/AlertBoxView.jsx

```diff
@@ -0,0 +1,69 @@
+import React, {PureComponent} from "react";
+import View from "./View";
+import {AlertBox, SegmentedControl} from "src";
+import Example, {ExampleCode, CodeSample} from "./Example";
+
+export default class AlertBoxView extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      type: "warning",
+    };
+  }
+
+  render() {
+    const {AlertBoxOptions} = AlertBoxView;
+    const {type} = this.state;
+
+    return (
+      <View title="AlertBox" sourcePath="src/AlertBox/AlertBox.jsx">
+        <p>
+          A set of alert boxes. N.B. this requires the app you use it in to have
```

Comment:
It would be better if it didn't, since it requires fontawesome assets which may not be available in the client app; so you can get the SVG versions of the icons, and copy them into components like so:

```
function MyIcon() {
  return <svg>...</svg>; // copy the svg contents here
}
```